### ISSUE TITLE
Multiple test build problems due to printf'ing std::clock_t

### ISSUE THREAD

**Issue by AMDmi3** [g-truc/glm#778]
FreeBSD 11.1, clang 4:

```
.../glm-0.9.9.0/test/core/core_func_matrix.cpp:300:71: error: format specifies type 'unsigned long' but the argument has type 'int' [-Werror,-Wformat]                                                                       
        printf("inverse<%s>(%f): %lu\n", Message, static_cast<double>(Diff), EndTime - StartTime);                                                                                                                                                                    
                                 ~~~                                         ^~~~~~~~~~~~~~~~~~~                                                                                                                                                                      
                                 %d                                                                                                                                                                                                                                   
```

This problem exists in
- test/gtc/gtc_integer.cpp
- test/core/core_type_vec4.cpp
- test/core/core_func_matrix.cpp
- test/gtx/gtx_fast_trigonometry.cpp

std::clock_t size is [unspecified](http://en.cppreference.com/w/cpp/chrono/c/clock_t), so passing it directly into printf is a bad idea. Either casts are needed, or printfs should be replaced with streams (which seems better and more C++ish solution to me).

**Comment by Groovounet** [comment]
This issue should be fixed in master branch for GLM 0.9.9.1 release.

Thanks for reporting,
Christophe

**Comment by AMDmi3** [comment]
Thanks!
